test/mix/tasks/phoenix.digest_test.exs

```diff
@@ -0,0 +1,34 @@
+Code.require_file "../../../installer/test/mix_helper.exs", __DIR__
+
+defmodule Mix.Tasks.Phoenix.DigestTest do
+  use ExUnit.Case
+
+  setup do
+    Mix.Task.clear
```

Comment:
We don't need this because you call the tasks directory and not via `Mix.Task.run/1`. This test case can even be async (async: true).